db/migrate/20200123221752_add_ercc_only_host_genomes.rb

```diff
@@ -0,0 +1,90 @@
+# This adds a bunch of hosts which users are known to have taken samples from.
+# They initially default to have ERCCs only subtracted.
+# See https://czi.quip.com/yPgpAPj1dtEC/Hosts-Dropdown-Options
+class AddErccOnlyHostGenomes < ActiveRecord::Migration[5.1]
+  HOST_GENOME_NAMES = [
+    'Badger',
+    'Bank Vole',
+    'Bat',
+    'Batfish',
+    'Bear',
+    'Bee',
+    'Carp',
+    'Cat',
+    'Camel',
+    'Chimpanzee',
+    'Cow',
+    'Dik Dik',
+    'Dog',
+    'Donkey',
+    'Chicken',
+    'Ferret',
+    'Field Vole',
+    'Fox',
+    'Giraffe',
+    'Goat',
+    'Gorilla',
+    'Guinea Pig',
+    'Harbor Seal',
+    'Hedgehog',
+    'Horse',
+    'Human',
+    'Kestrel',
+    'Leopard',
+    'Lion',
+    'Llama',
+    'Mallard',
+    'Milu',
+    'Monk Seal',
+    'Monkey',
+    'Mosquito',
+    'Mouse',
+    'Owl',
+    'Penguin',
+    'Pig',
+    'Porpoise',
+    'Prairie Dog',
+    'Raccoon',
+    'Rabbit',
+    'Rat',
+    'Sea Lion',
+    'Seagull',
+    'Seal',
+    'Seastar',
+    'Sloth',
+    'Snake',
+    'Stork',
+    'Taurine Cattle',
+    'Tick',
+    'Tiger',
+    'Turkey',
+    'Unknown',
+    'Water Buffalo',
```

Comment:
it looks like there are some missing from here / the doc that we have already like `Salpingoeca rosetta` or `Torafugu` https://idseq.net/host_genomes ? Is that OK?

Reply:
@oliviabholmes ? @katrinakalantar ?

FWIW, the migration should skip over existing. ```    return if HostGenome.find_by(name: name)
```